src/microbe_stage/PlacedOrganelle.cs

```diff
@@ -198,7 +197,6 @@ public void OnAddedToMicrobe(Microbe microbe)
     {
         if (Definition == null)
             throw new Exception("PlacedOrganelle has no definition set");
-
```

Comment:
Same here, from what I recall the rest of the code separates such `if`s by a line break.